core/src/main/scala/io/finch/response/package.scala

```diff
@@ -151,25 +152,28 @@ package object response {
    * An abstraction that is responsible for encoding the response of type `A`.
    */
   trait EncodeResponse[-A] {
-    def apply(rep: A): String
+    def apply(rep: A): Array[Byte]
     def contentType: String
+    def charset: Option[String] = None
```

Comment:
I will. And I'm going to add a documentation that when treating binary contents, charset must be set `None` explicitly.